vizro-core/docs/pages/user_guides/run.md

```diff
@@ -59,9 +59,17 @@ The dashboard application can be launched in jupyter environment in `inline`, `e
 - you can specify `jupyter_mode="external"` and a link will be displayed to direct you to the localhost where the dashboard is running.
 - you can use tab mode by `jupyter_mode="tab"` to automatically open the app in a new browser
 
-## Launch it via Gunicorn
+## Deployment of Vizro app
```

Comment:
To be very accurate, this instruction is still more focus on "running the app with gunicorn" than how to deploy the app 🤔 

Shall we phrase it as something like, for deployment, it's recommended to run the app with gunicorn, and here is how

Reply:
Agreed, in the current form, this point breaks the reading flow. This entire guide has the overarching title of "How to launch the dashboard".

I was going to suggest to write a separate section on how to deploy the dashboard, when I realised this is exactly what we had 😅 .

I would suggest keeping the old structure, but to maybe enhance it with the additional links and so on.

Overall then I think the title of this section should change back to "Gunicorn", and ideally the one above to simply "Jupyter Environment". Then the sections alrady contain the answer to the user guide question, which I think is an improvement.

Reply:
See what you think of the new version 🙂 I even did the special popup thing:
![image](https://github.com/mckinsey/vizro/assets/49395058/f78b88af-8e41-4345-a31c-0762a3a8f994)